_data/tools.yml

```diff
@@ -339,6 +339,17 @@
   description: OpenAPI-GUI is a GUI for creating and editing OpenAPI version 3.0.x JSON/YAML definitions.
   v2: false
   v3_progress_link: https://github.com/Mermade/openapi-gui/blob/master/docs/openapi3-support.md
+  
+- name: Hackolade
+  category: gui-editors
+  language: ReactJS
+  link: https://hackolade.com
+  github: https://github.com/hackolade/OpenAPI
+  description: |
+    A schema-centric visual editor for Swagger and OpenAPI, from the pioneer in data modeling for NoSQL databases.
```

Comment:
can you remove Swagger here? The docs state you need a plugin, and that this supports just OAS 3.0.

Reply:
The tool is also available for Swagger.  I updated our OAS 3.0 readme with a link to our Swagger 2 plugin at the top.  Hopefully that works for you.  Please let me know otherwise.